dashboard/app/helpers/users_helper.rb

```diff
@@ -12,6 +12,29 @@ module UsersHelper
   ACCT_TAKEOVER_OAUTH_TOKEN = 'clever_takeover_token'
   ACCT_TAKEOVER_FORCE_TAKEOVER = 'force_clever_takeover'
 
+  # Move followed sections from old_user to new_user and destroy old_user
+  def move_sections_and_destroy_old_user(old_user, new_user)
```

Comment:
The names here are a bit confusing, because in a typical use case the `new_user` is the older, existing account, and `old_user` is the newer, temporary account (unless I'm misreading this).  Named parameters would help a bit.  Changing  from old/new to something like source/destination might be better.

Reply:
ah, yeah, great idea. i'll make that update